corehq/apps/geospatial/static/geospatial/js/gps_capture.js

```diff
@@ -219,6 +227,42 @@ hqDefine("geospatial/js/gps_capture",[
             self.isCreatingCase(true);
             const caseToCreate = new dataItemModel(null, self.dataType);
             self.captureLocationForItem(caseToCreate);
+
+            $("#owner-select").select2({
+                placeholder: gettext('Current User'),
```

Comment:
It would be nice to show the current owner's username + indicate that it is the current user. Something like 
```
<username> (current owner)
```

Not sure how easy this would be to achieve, since `urrent User` is the placeholder..?

Reply:
I like this idea, it makes it more clear who the current user is. It should be fairly straightforward to achieve, I'll just need to pass the current user along in the context parameters so that the front-end has access to it.

Reply:
[196ded3](https://github.com/dimagi/commcare-hq/pull/33791/commits/196ded33146993baf46533968b70fe5aac466c80)